flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+    "only-arches": ["x86_64"]
```

Comment:
Any reason for this being x86_64 only?

Reply:
I can only test it on this arch. Which other ones would be relevant?